app.py

```diff
@@ -12,8 +12,23 @@
 from src.price import update_price
 from src.rates import plot_rates, rates_table
 
-# Constant for default pricing date
-DEFAULT_PRICING_DATE = datetime(2023, 12, 31)
+# Default pricing date and rates
+DEFAULT_PRICING_DATE = datetime(2024, 1, 2)
+DEFAULT_RATE_DATA = [
```

Comment:
can't we use something like,
get_rates_for_date(DEFAULT_PRICING_DATE)
instead of hardcoding the data here?